Content.Shared/Doors/Systems/SharedFirelockSystem.cs

```diff
@@ -61,14 +60,6 @@ private void OnBeforeDoorOpened(EntityUid uid, FirelockComponent component, Befo
             WarnPlayer((uid, component), args.User.Value);
     }
 
-    private void OnBeforePry(EntityUid uid, FirelockComponent component, ref BeforePryEvent args)
-    {
-        if (args.Cancelled || !component.Powered || args.StrongPry || args.PryPowered)
-            return;
-
-        args.Cancelled = true;
```

Comment:
as far as I can tell literally just comment out this one line and Everything Will Be Fine or some shit